applications/FluidDynamicsApplication/custom_elements/vms_adjoint_element.h

```diff
@@ -238,7 +238,7 @@ class KRATOS_API(FLUID_DYNAMICS_APPLICATION) VMSAdjointElement: public Element {
      *
      * @return 0 after successful completion.
      */
-    int Check(const ProcessInfo &/*rCurrentProcessInfo*/) override
+    int Check(const ProcessInfo &/*rCurrentProcessInfo*/) const override
```

Comment:
this should be implemented, please check the function

Reply:
Agree... This is awkward. I will do it on here.